### PR TITLE
feat : Profile page and Toast message

### PR DESCRIPTION
**Pull request by anuj-kumary**
This PR includes:-

- Profile Page
- Toast Message
- Copy Link Video

Preview:- https://deploy-preview-8--techhub-react.netlify.app/

### CODE REVIEW

File: src/pages/Profile/Profile.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.profile__card {
+    justify-content: center;
+    align-items: center;
+}
+
+.line {
+    border-bottom: 1px solid #fff;
+    padding: 1rem .6rem;
+}
```
Comment (samyakshah3008) [inline src/pages/Profile/Profile.css:9]: Can use CSS variables here

**Review by samyakshah3008 — APPROVED** [review]
Added small suggestion 
Good documentation 
Good for approval ✅

File: src/pages/Profile/Profile.css (hunk 1 above)
Comment (anuj-kumary) [reply src/pages/Profile/Profile.css:9]: Done

**Review by Nishant-96 — APPROVED** [review]

**Comment by Nishant-96** [comment]
Looks good.
- good naming conventions followed.
- Liked the toast Didnt knew about react toastify.

**Pull request merged**